core/src/main/java/io/jenkins/pluginhealth/scoring/probes/KnownSecurityVulnerabilityProbe.java

```diff
@@ -55,7 +56,7 @@ protected ProbeResult doApply(Plugin plugin, ProbeContext context) {
                 final Matcher matcher = pattern.matcher(plugin.getVersion().toString());
                 return matcher.find();
             }))
-            .map(SecurityWarning::id)
+            .map(SecurityWarning::url)
```

Comment:
see if you can map it to a structure to store both `id` and `url`.

Reply:
@alecharp Do we need to show the front-end part like this ?

SECURITY-1511  <URL>

Reply:
It would be great to be able to show the ID as an hyperlink yes.

Reply:
> It would be great to be able to show the ID as an hyperlink yes.

Thanks that means 
SECURITY-1511 [SECURITY-1511](https://jenkins.io/security/advisory/2020-03-09/#SECURITY-1511)